commands/dns_cli.py

```diff
@@ -266,7 +287,7 @@ def create_hosted_zone(client, domain, start_domain, base_len):
             },
         )
 
-        return True
+    return True
```

Comment:
This was causing the function to return after doing the first hosted zone, which was problematic if more than one part of the chain needs to be created.